content/docs/gherkin/reference.md

```diff
@@ -303,6 +306,11 @@ Feature: Multiple site support
     Then I should see "Your article was published."
 ```
 
+You can only have one set of `Background` steps at the `Feature` level. If you need different `Background` steps for different scenarios, you have a couple of options:
```

Comment:
What about a Background at Rule level? Can we have multiple Rules in one feature file? If so, can we have multiple backgrounds in that file? How would that work (i.e. which will run when and in what order)? Some examples would be nice. They don't have to be here; it could also be a link if we already have examples.

Reply:
Good point. In my testing on `cucumber-js`, given this feature:

```gherkin
Feature: a feature
  Background:
    Given some feature-level setup

  Rule:
    Background:
      Given some rule-level setup

    Example:
      When I do a thing
```

The resulting pickle will be:

```gherkin
Given some feature-level setup
Given some rule-level setup
When I do a thing
```

So before hooks, then feature-level background, then rule-level background, then scenario/example steps. I'm assuming that is the intent...

Reply:
Pushed an update that includes:

- An example of usage of `Background` within `Rule`
- Some clearer wording around placement
- An explanation about using both `Feature` and `Rule` level together